spec/lib/simp/cli/config/items/action/files/answers_yaml_file_writer.yaml

```diff
@@ -2,7 +2,7 @@
 # simp config answers
 #
 # Generated for simp-lite scenario on 2017-01-13 11:42:03
-# using simp-cli version 2.0.0
+# using simp-cli version 2.0.1
```

Comment:
In another PR, I've fixed the test so you don't have to keep fixing these files.